### ISSUE TITLE
Conversion stuck in 100%

### ISSUE THREAD

**Issue by olivatooo** [NFJones/audio-to-midi#3]

**Comment by rajdeepbharati** [comment]
I'm also facing the same issue

**Comment by shoegazerstella** [comment]
Same issue also here. Gets stuck at 100% and the generated .mid file seems to be corrupted

**Comment by ttuanho** [comment]
> Same issue also here. Gets stuck at 100% and the generated .mid file seems to be corrupted

In my case, the `.mid` files are totally empty.

**Comment by markhanslip** [comment]
> > Same issue also here. Gets stuck at 100% and the generated .mid file seems to be corrupted
> 
> In my case, the `.mid` files are totally empty.

same here

**Comment by ttuanho** [comment]
Apparently, according to [this](https://stackoverflow.com/a/5852139/10643419), converting audio to midi may not actually every work well.

There's also another suggestion of *WaoN* in that thread.

**Comment by shoegazerstella** [comment]
I am using [this](https://github.com/trevorwelch/chroma-to-midi). works nicely

**Comment by shoegazerstella** [comment]
> Apparently, according to [this](https://stackoverflow.com/a/5852139/10643419), converting audio to midi may not actually every work well.
> 
> There's also another suggestion of _WaoN_ in that thread.

what I am doing is using a music source separation network and convert to midi each resulting stem.

**Comment by olivatooo** [comment]
@ttuanho Yeah, don't make sense to get the original song in midi. But it isn't the goal here. i.e can see the @shoegazerstella solution, it's good one.

**Comment by olivatooo** [comment]
@shoegazerstella Thanks for sharing here!! `chroma-to-midi` I'll give try

**Comment by yuvraj108c** [comment]
same issue
